docs/setup/independent/install-kubeadm.md

```diff
@@ -132,6 +132,11 @@ You will install these packages on all of your machines:
 
 * `kubectl`: the command line util to talk to your cluster.
 
+kubeadm **will not** install or manage `kubelet` or `kubectl` for you, so you will 
+need to ensure they match the version of the Kubernetes control panel you want 
+kubeadm to install for you. If you do not, there is a risk of a version skew occurring that
```

Comment:
However: one minor version skew between the kubelet and the control plane / kubeadm **is** supported. e.g. 1.7.0 kubelets, 1.8.0 API server + kubeadm is totally ok. Not the other way round though. Please send a new PR with that; also add this to the version skew policy section of the main doc. Thanks!